VMEncryption/test/test_check_util.py

```diff
@@ -121,17 +121,46 @@ def test_mount_scheme(self):
         with mock.patch("__builtin__.open", mock.mock_open(read_data=proc_mounts_output)) as mock_open:
             self.assertFalse(self.cutil.is_unsupported_mount_scheme())
 
-    @mock.patch("os.system", return_value=0)
-    def test_lvm_os_lvm_present(self, os_system):
-        # if there is LVM
-        self.assertFalse(self.cutil.is_invalid_lvm_os())
+    # Skip LVM OS validation when OS volume is not being targeted
+    def test_skip_lvm_os_check_if_data_only_enable(self):
+        # skip lvm detection if data only 
+        self.cutil.validate_lvm_os({Common.CommonVariables.VolumeTypeKey: "DATA", Common.CommonVariables.EncryptionEncryptionOperationKey: Common.CommonVariables.EnableEncryption})
+
+    def test_skip_lvm_os_check_if_data_only_ef(self):
+        # skip lvm detection if data only 
+        self.cutil.validate_lvm_os({Common.CommonVariables.VolumeTypeKey: "DATA", Common.CommonVariables.EncryptionEncryptionOperationKey: Common.CommonVariables.EnableEncryptionFormat})
+
+    def test_skip_lvm_os_check_if_data_only_efa(self):
+        # skip lvm detection if data only 
+        self.cutil.validate_lvm_os({Common.CommonVariables.VolumeTypeKey: "DATA", Common.CommonVariables.EncryptionEncryptionOperationKey: Common.CommonVariables.EnableEncryptionFormatAll})
+
+    def test_skip_lvm_os_check_if_data_only_disable(self):
+        # skip lvm detection if data only 
+        self.cutil.validate_lvm_os({Common.CommonVariables.VolumeTypeKey: "DATA", Common.CommonVariables.EncryptionEncryptionOperationKey: Common.CommonVariables.DisableEncryption})
+
+    def test_skip_lvm_os_check_if_query_only_data(self):
```

Comment:
QueryEncryptionStatus generally is not accompanied by any volume type.
Please add a test case for that scenario.